receiver/gitproviderreceiver/internal/scraper/githubscraper/helpers.go

```diff
@@ -311,19 +312,78 @@ func getAge(start time.Time, end time.Time) int64 {
 
 func (ghs *githubScraper) getCVEs(
 	ctx context.Context,
-	client graphql.Client,
+	gClient graphql.Client,
+	rClient *github.Client,
 	repo string,
 ) (map[metadata.AttributeCveSeverity]int64, error) {
-	c, err := getRepoCVEs(ctx, client, ghs.cfg.GitHubOrg, repo)
-	if err != nil {
-		return nil, err
+	d := ghs.getDepBotAlerts(ctx, gClient, repo)
+	c := ghs.getCodeScanAlerts(ctx, rClient, repo)
+
+	return mapSeverities(d, c), nil
+}
+
+func (ghs *githubScraper) getDepBotAlerts(
+	ctx context.Context,
+	gClient graphql.Client,
+	repo string,
+) []CVENode {
+
+	var all []CVENode
```

Comment:
I think we talked about setting this to `alerts` and on 335 setting that to `a`

Reply:
I think it was the code scanning function where we discussed this. Regardless I'll make a change so this function can read similarly to that.

Reply:
Made the changes. Marking as resolved